refnx/reflect/_app/test/test_app.py

```diff
@@ -83,16 +82,16 @@ def test_add_spline_save(qtbot, tmpdir):
 
 def save_and_reload_experiment(app, tmpdir):
     # save and reopen experiment.
-    sf = pjoin(str(tmpdir), "experiment1.mtft")
+    sf = Path(tmpdir) / "experiment1.mtft"
     # this is just to make sure that the file exists
     with open(sf, "wb") as f:
         f.write(b"sksij")
 
-    app.settings.experiment_file_name = sf
+    app.settings.experiment_file_name = str(sf)
```

Comment:
Is it worth broadening that property to accept Path objects directly?

Reply:
Not really, but I've just made it insensitive.